docarray/array/array/array.py

```diff
@@ -136,10 +135,27 @@ def __init__(
         docs: Optional[Iterable[T_doc]] = None,
         tensor_type: Type['AbstractTensor'] = NdArray,
     ):
-
         self._data: List[T_doc] = list(self._validate_docs(docs)) if docs else []
         self.tensor_type = tensor_type
 
+    @classmethod
+    def construct(
+        cls: Type[T],
+        docs: Sequence[T_doc],
+        tensor_type: Type['AbstractTensor'] = NdArray,
+    ) -> T:
+        """
+        Create a DocumentArray without validation any data. The data must come from a
```

Comment:
When would validation happen? In the `parse_obj_as`? So are we extendind pydantic-style validation to non-pydantic classes?

Reply:
> When would validation happen? In the parse_obj_as? So are we extendind pydantic-style validation to non-pydantic classes?

yes we expand pydantic validation to array